tests/Fixer/ClassNotation/FinalClassFixerTest.php

```diff
@@ -37,13 +37,20 @@ public static function provideFixCases(): array
     {
         return [
             ['<?php /** @Entity */ class MyEntity {}'],
+            ['<?php #[Entity] class MyEntity {}'],
             ['<?php use Doctrine\ORM\Mapping as ORM; /** @ORM\Entity */ class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; #[ORM\Entity] class MyEntity {}'],
             ['<?php use Doctrine\ORM\Mapping; /** @Mapping\Entity */ class MyEntity {}'],
             ['<?php use Doctrine\ORM; /** @ORM\Mapping\Entity */ class MyEntity {}'],
+            ['<?php use Doctrine\ORM; #[ORM\Mapping\Entity] class MyEntity {}'],
             ['<?php /** @Document */ class MyDocument {}'],
+            ['<?php #[Document] class MyDocument {}'],
             ['<?php use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM; /** @ODM\Document */ class MyEntity {}'],
+            ['<?php use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM; #[ODM\Document] class MyEntity {}'],
             ['<?php /** @entity */ class MyEntity {}'],
-            ['<?php use Doctrine\ORM\Mapping as ORM; /** @orm\entity */ class MyEntity {}'],
+            ['<?php #[Entity] class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; /** @ORM\entity */ class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; #[ORM\entity] class MyEntity {}'],
```

Comment:
Cases with attributes must be moved to separate test/provider and run only on PHP >= 8.0 - on 7.4 it is treated as comment, so effectively the code is like `<?php use Doctrine\ORM\Mapping as ORM;` and I believe such test does not provide value.

Just look for `testFix80` and `provideFix80Cases` in the project to see how it's done.